packages/frames.js/src/core/createFrames.test.ts

```diff
@@ -137,4 +137,85 @@ describe("createFrames", () => {
 
     expect(response).toBeInstanceOf(Response);
   });
+
```

Comment:
what happens when baseUrl includes a path and a basePath is also specified?

Reply:
will be resolved relatively to url with path. Will add a test for it

Reply:
Ok that's actually not true, it will override the path.

Reply:
`baseUrl: 'http://test.com/test', basePath: '/override'` will produce `http://test.com/override`

Reply:
this is unexpected behaviour no?

Reply:
I think we should keep the logic as is and just change tsdoc to reflect that the basePath will override the path of `baseUrl`.

Reply:
alright